src/indexes.ts

```diff
@@ -568,6 +569,45 @@ class Index<T = Record<string, any>> {
     return task
   }
 
+  ///
+  /// PAGINATION SETTINGS
+  ///
+
+  /**
+   * Get the pagination settings
+   * @memberof Index
+   * @method getPagination
+   * @returns {Promise<object>} Promise containing object of pagination settings
```

Comment:
```suggestion
   * @returns {Promise<object>} Promise containing object of pagination settings
```

This should not return an `<object>` but the type your created

Reply:
Return `PaginationSettings`